python/create_graph_data.py

```diff
@@ -0,0 +1,85 @@
+import pandas as pd
```

Comment:
Maybe we just have one graph_utils.py file and combine all the utils? seems like these are both utils